### PR TITLE
Feat homepage

### PR DESCRIPTION
**Pull request by abhij1607**
image

- design system CSS
- Nav
- Home page UI
- categories-context
- categories in nav
- Footer

### CODE REVIEW

File: src/context/categories-context.jsx [hunk 1]
```diff
@@ -0,0 +1,27 @@
+import axios from "axios";
+import { useState, createContext, useContext, useEffect } from "react";
+import { Categories } from "../pages/Home/categories";
+
+// const fetchCategories = async () => {
```
Comment (shivampandey0) [inline src/context/categories-context.jsx]: remove unused code comments

**Review by shivampandey0 — COMMENTED** [review]
Code looks good and easy to understand with components for pages.
- Remove unnecessary comments

File: src/context/categories-context.jsx (hunk 1 above)
Comment (abhij1607) [reply src/context/categories-context.jsx]: sure, thanks for pointing it out

**Review by GuptaEesh — COMMENTED** [review]
Code looks structured.
good folder structure.
Can use props for direct manipulation of components

File: src/components/NavBar/nav-categories.js [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+const NavCategories = () => {
+  const { categories } = useCategories();
+  return (
+    <nav className="pdt-category-bar box-shadow navigation">
+      <ul className="align-center list-structure">
+        {categories.map((category) => {
+          return (
+            <li key={category._id} className="list-item-inline">
+              <Link className="link" to="./product.html">
```
Comment (bajajcodes) [inline src/components/NavBar/nav-categories.js:12]: This might be some typo:)

**Review by bajajcodes — COMMENTED** [review]
- Consistency is maintained in code writing style.
- For hard-coded values such as image source can use static data file.
- Found value of to property as `./file-name.html` for Links in `top-nav.js`.

**Pull request merged**